### ISSUE TITLE
segfault on import caffe

### ISSUE THREAD

**Issue by cyphunk** [WiMLDS/deepdream-workshop#2]
I guess I should report this somewhere else. But perhaps if by chance you ran into this error:

I ran the notebook and was able to import caffe and even build a dream image from the example clouds but it then failed on something. I quickly went back to this tutorial and realized i forgot:

```
cd python
for req in $(cat requirements.txt); do pip install $req; done
```

After installing these modules caffe then segfaults

```
$ python
Python 2.7.9 (default, Jan  7 2015, 11:49:12) 
[GCC 4.2.1 Compatible Apple LLVM 6.0 (clang-600.0.56)] on darwin
>>> import caffe
Segmentation fault: 11
```

**Comment by cyphunk** [comment]
Appears I have set the PYTHON_LIB directory incorrectly in Makefile.config. Corrected to `/usr/local/Cellar/python/2.7.9/Frameworks/Python.framework/Versions/2.7/lib` and it worked.

**Comment by ledell** [comment]
Thanks for your report and follow-up solution.
